src/styles/styles.scss

```diff
@@ -32,15 +32,23 @@ h1,h2
   display: flex;
   justify-content: center;
 }
-
+.submit
```

Comment:
The class declaration for ".topnav" follows a different format by being on the same line as the opening {. These should match.